src/SI_Toolkit/computation_library.py

```diff
@@ -54,7 +54,7 @@ class ComputationLibrary:
     gather: Callable[[TensorType, TensorType, int], TensorType] = None
     gather_last: Callable[[TensorType, TensorType], TensorType] = None
     arange: Callable[[Optional[NumericType], NumericType, Optional[NumericType]], TensorType] = None
-    zeros: Callable[["tuple[int]"], TensorType] = None
+    zeros: Callable[["tuple[int,...]"], TensorType] = None
```

Comment:
Very nice! I had wrong assumptions about how to type subscript tuples. Learned something again.